test/models/docker_builder_service_test.rb

```diff
@@ -1,6 +1,6 @@
 require_relative '../test_helper'
 
-SingleCov.covered! uncovered: 19
+SingleCov.covered! uncovered: 20
```

Comment:
nah ... just sanity checking whenever this increases ... you are welcome to
test it, but that can be a separate PR :)

On Wed, Jul 13, 2016 at 4:19 PM, Tran Hai Dang <notifications@github.com>
wrote:

> In test/models/docker_builder_service_test.rb
> <https://github.com/zendesk/samson/pull/1076#discussion_r70725527>:
>
> > @@ -1,6 +1,6 @@
> >  require_relative '../test_helper'
> >
> > -SingleCov.covered! uncovered: 19
> > +SingleCov.covered! uncovered: 20
>
> The run! method's body is uncovered and originally had 19 lines. I added
> an if check for running the remote job and now the body has one extra line.
> Do you have any idea how to test the code block passed to the job execution
> inside this method's body?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendesk/samson/pull/1076/files/3dca884387016cc9f2b1ffd3594c6cfbe9d9a34b#r70725527>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAAsZwX8mCKykD3mFr1AtS1C-6hu2i5Gks5qVXKHgaJpZM4I2Rcw>
> .
>